cypress/e2e/ui/Settings/Application-Settings/schedule.cy.js

```diff
@@ -0,0 +1,348 @@
+    cy.get('label[for="target_id"]').should('exist');
+    cy.get('input[role="combobox"]#target_id').should('exist');
+
+    // Checking for Attribute/Value pairs
+    cy.contains('h3', 'Attribute/Value Pairs').should('exist');
+    // Checking for 5 attribute-value pairs text fields
+    cy.get('input#attribute_1').should('exist');
+    cy.get('input#value_1').should('exist');
+    cy.get('input#attribute_2').should('exist');
+    cy.get('input#value_2').should('exist');
+    cy.get('input#attribute_3').should('exist');
+    cy.get('input#value_3').should('exist');
+    cy.get('input#attribute_4').should('exist');
+    cy.get('input#value_4').should('exist');
+    cy.get('input#attribute_5').should('exist');
+    cy.get('input#value_5').should('exist');
+
+    /* ===== Selecting "Once" option from "Run" dropdown does not show the "Every" dropdown ===== */
+
+    cy.get('select#timer_typ').select('Once');
+    // Checking whether the Every dropdown is hidden
+    cy.get('input#timer_value').should('not.exist');
+
+    /* ===== Selecting any other option other than "Once" from "Run" dropdown shows the "Every" dropdown ===== */
+
+    cy.get('select#timer_typ').select('Hours', { force: true });
+    // Checking whether the "Every" dropdown exist
+    cy.get('label[for="timer_value"]').should('exist');
+    cy.get('select#timer_value').should('exist');
+
+    cy.get('select#timer_typ').select('Days', { force: true });
+    // Checking whether the "Every" dropdown exist
+    cy.get('label[for="timer_value"]').should('exist');
+    cy.get('select#timer_value').should('exist');
+
+    cy.get('select#timer_typ').select('Weeks', { force: true });
+    // Checking whether the "Every" dropdown exist
+    cy.get('label[for="timer_value"]').should('exist');
+    cy.get('select#timer_value').should('exist');
+
+    cy.get('select#timer_typ').select('Months', { force: true });
+    // Checking whether the "Every" dropdown exist
+    cy.get('label[for="timer_value"]').should('exist');
+    cy.get('select#timer_value').should('exist');
+  });
+
+  it('Checking whether Cancel button works on the Add form', () => {
+    selectConfigMenu();
+    cy.contains('#main-content .bx--btn-set button[type="button"]', 'Cancel')
+      .should('be.enabled')
+      .click();
+    cy.get('#main_div #flash_msg_div .alert-success').contains(
+      'Add was cancelled by the user'
+    );
+  });
+
+  it('Checking whether add, edit & delete schedule works', () => {
+    /* ===== Adding a schedule ===== */
+    addSchedule();
+    cy.get('#main_div #flash_msg_div .alert-success').contains(
+      'Schedule "Test name" was saved'
+    );
+
+    /* ===== Editing a schedule ===== */
+    // Selecting the created schedule
+    cy.contains('li.list-group-item', 'Test name').click();
+    selectConfigMenu('Edit this Schedule');
+    // Editing name and description
+    cy.get('input#name').clear().type('Dummy name');
+    cy.get('input#description').clear().type('Dummy description');
```

Comment:
note, I found that our code does ajax calls to form changed or form field changed each time we type.  I needed to fix an issue in the report pages by waiting for these ajax calls to complete before moving to the next field. See: https://github.com/ManageIQ/manageiq-ui-classic/pull/9487